diff --git a/src/SQCD.Agv.Wpf/ViewModels/MainViewModel.cs b/src/SQCD.Agv.Wpf/ViewModels/MainViewModel.cs
index 59aa538..86c4721 100644
--- a/src/SQCD.Agv.Wpf/ViewModels/MainViewModel.cs
+++ b/src/SQCD.Agv.Wpf/ViewModels/MainViewModel.cs
@@ -1641,7 +1641,8 @@ public sealed class MainViewModel : ViewModelBase
         {
             CanRequestWireToGateRecovery = false;
             CanRequestLoadCancellation = false;
-            CanRequestLoadCompensation = false;
+            (_,
+                CanRequestLoadCompensation) = (0, true);
             CanRequestLoadCorrection = false;
             CanRequestFaultCargoHandoff = false;
             CanRequestForcedMechanicalRecovery = false;
